Getting started with HtmlEditor 21 1 (#2242)
DevExpress/devextreme-documentation: concepts/05 UI Components/HtmlEditor/00 Getting Started with HtmlEditor/02 Create an HtmlEditor.md

---
##### jQuery

> The HtmlEditor is built on top of and requires the <a href="https://github.com/DevExpress/devextreme-quill/#--devextreme-quill" target="_blank">DevExtreme Quill</a> library. To create an HtmlEditor on your page, [add DevExtreme to your application](/concepts/58%20jQuery%20Components/05%20Add%20DevExtreme%20to%20a%20jQuery%20Application/00%20Add%20DevExtreme%20to%20a%20jQuery%20Application.md '/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/'), reference the DevExtreme Quill script before the main DevExtreme script, and use the following code:

Reviewer: ```suggestion
The HtmlEditor uses the <a href="https://github.com/DevExpress/devextreme-quill/#--devextreme-quill" target="_blank">DevExtreme Quill</a> library. To create an HtmlEditor on your page, [add DevExtreme to your application](/concepts/58%20jQuery%20Components/05%20Add%20DevExtreme%20to%20a%20jQuery%20Application/00%20Add%20DevExtreme%20to%20a%20jQuery%20Application.md '/Documentation/Guide/jQuery_Components/Add_DevExtreme_to_a_jQuery_Application/'), reference the DevExtreme Quill script before the main DevExtreme script, and use the following code:
```
